vars/testsInStage.groovy

```diff
@@ -25,6 +25,11 @@ boolean call() {
                              exit 0
                          fi
                          cd src/tests/ftest
-                         ./list_tests.py """ + parseStageInfo()['test_tag'],
+                         if [ -f list_tests.py ]
```

Comment:
```suggestion
                         if [ -e list_tests.py ]
```

It has to be executable, not merely exist, to be run without specifying an interpreter on the command line.

Reply:
Done, all working.